content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md

```diff
@@ -10,45 +10,160 @@ content_template: templates/concept
 
 {{% capture overview %}}
 
-This document describes how to set up TLS client certificate bootstrapping for
-kubelets. Kubernetes 1.4 introduced an API for requesting certificates from a
-cluster-level Certificate Authority (CA). The original intent of this API is to
-enable provisioning of TLS client certificates for kubelets. The proposal can be
+In a kubernetes cluster, the components on the worker nodes - kubelet and kube-proxy - need to communicate with Kubernetes master components, i.e. kube-apiserver.
+In order to ensure that communication is kept private, not interfered with, and ensure that each is talking to a trusted element, we strongly
```

Comment:
"that each is talking to a trusted " -> Seems to be missing a word..
Did you mean the following: "that each cluster is talking to a trusted"

Reply:
Yeah, will fix that too. Updated commit coming momentarily.

Reply:
new commit out. I didn't squash them to make it easier for you to see changes.